src/plopp/backends/pythreejs/canvas.py

```diff
@@ -64,6 +64,7 @@ def __init__(
         )
 
     def to_widget(self):
+        self.renderer.layout = ipw.Layout(max_width='80%', overflow='auto')
```

Comment:
The fix actually work very nicely, and as you say is non-invasive.

Any reason why you set the layout every time `to_widget` is called (which is when the repr for the canvas is requested) as opposed to passing the layout once to the `Renderer` constructor above?

Reply:
I just want to localize this as much as possible as I don't really like this hard coded behaviour. We only deal with this problem (right now) when someone tries to bound the canvas generated next to some other widget layout. The `to_widget` call seems like an explicit point where the underlying canvas will interact with other widgets.